### ISSUE TITLE
Investigate using larger student models

### ISSUE THREAD

**Issue by marco-c** [mozilla/firefox-translations-training#174]
Current models are around ~20 MB, this is very little for Desktop. We could try to go with larger models and see what kind of quality improvements we manage to get.

**Comment by eu9ene** [comment]
I will try training the student Large or Base configurations from https://aclanthology.org/D19-5632.pdf. 


<img width="1049" alt="Screenshot 2024-10-10 at 11 58 31 AM" src="https://github.com/user-attachments/assets/c0da09e9-e1a8-4714-a932-5d911fb65c0a">

**Comment by eu9ene** [comment]
I looked at the recommended configurations in https://github.com/browsermt/students/tree/master/train-student/models and what HPLT folks are training here https://github.com/hplt-project/bitextor-mt-models/tree/main and it seems the recommended approach to train a larger model is to go with the Base configuration. The difference I see in code is basically these two model parameters:

base
```yaml
    dim-emb: 512
    transformer-dim-ffn: 2048
```
tiny
```yaml
    dim-emb: 256
    transformer-dim-ffn: 1536
```

**Comment by eu9ene** [comment]
It can give ~2 BLEU points bump and is twice slower, for example here https://github.com/hplt-project/bitextor-mt-models/tree/main/swa-eng.

I launched training for en-ru student in Base configuration: 
https://firefox-ci-tc.services.mozilla.com/tasks/P8eqvTKvRp640O8mcMZtWw/runs/0
https://wandb.ai/moz-translations/en-ru/runs/mjnbhst5?nw=nwuserepavlov

**Comment by eu9ene** [comment]
It looks pretty good so far! The gap is significantly smaller on the validation set now.

<img width="1352" alt="Screenshot 2024-10-11 at 9 34 44 AM" src="https://github.com/user-attachments/assets/a7a5a2c0-95bc-4135-a34e-6292db2e305e">

**Comment by gregtatum** [comment]
For clarity here, since I was confused for a bit, the `tiny` and `base` don't quite match up with the paper, as the paper's `base` model also increases the decoder depth. So here the terms have slightly different meanings.

**Comment by gregtatum** [comment]
(I updated the [summary comment](https://github.com/mozilla/firefox-translations-training/issues/174#issuecomment-2414864806) for some clarity on the result)
